src/routes/EResourceViewRoute.js

```diff
@@ -211,18 +228,21 @@ class EResourceViewRoute extends React.Component {
       <View
         key={get(resources, 'eresource.loadedAt', 'loading')}
         data={{
-          eresource: get(resources, 'eresource.records[0]', {}),
+          eresource: resources?.eresource?.records?.[0] ?? {},
           entitlementOptions: this.getRecords('entitlementOptions'),
           entitlements: this.getRecords('entitlements'),
           packageContentsFilter: this.props.resources.packageContentsFilter,
           packageContents: this.getPackageContentsRecords(),
           packageContentsCount: this.getPackageContentsRecordsCount(),
+          relatedEntitlements: this.getRecords('relatedEntitlements'),
         }}
         handlers={{
           ...handlers,
           onFilterPackageContents: this.handleFilterPackageContents,
           onNeedMorePackageContents: this.handleNeedMorePackageContents,
           onClose: this.handleClose,
+          onEdit: this.handleEdit,
+          onEresourceClick: this.handleEresourceClick,
```

Comment:
Lets avoid introducing a third way of casing "eresource". 😆 Just use `*EResourceClick` here.